src/test/scala/htsjdk/UnitSpec.scala

```diff
@@ -0,0 +1,6 @@
+package htsjdk
+
+import org.scalatest.{FlatSpec, Matchers}
+
+/** Base class for all Scala tests. */
+class UnitSpec extends FlatSpec with Matchers
```

Comment:
why "UnitSpec"? if the java class is HtsjdkTest, shouldn't the scala version be close to that? perhaps HtsjdkScalaTest? Or at least HtsjdkSpec...